.github/workflows/codeql.yml

```diff
@@ -12,7 +12,7 @@ permissions:
 jobs:
   analyze-ubuntu:
     name: Analyze on Ubuntu
-    runs-on: ubuntu-latest
+    runs-on: ${{ github.repository_owner == 'intel' && 'intel-ubuntu-latest' || 'ubuntu-latest' }}
```

Comment:
I believe our repo's owner is `oneapi-src`

Reply:
that's what I thought, but wasn't sure :D done.